openshift/certbot/openshift/certbot.bc.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: v1
```

Comment:
Should this be `template.openshift.io/v1`?

Reply:
Again - it being directly from the certbot project I'd rather leave it alone. The build config is only run once, manually - so if it has that warning it doesn't affect us.